tests/test_move_embargo_args.py

```diff
@@ -1,39 +1,289 @@
+import subprocess
 import unittest
-# import sys
-# sys.path.insert(0, "..")
-# from src import move_embargo_args
-import move_embargo_args
-
-
-class TestEmbargoArgs(unittest.TestCase):
-    def test_parser(self):
-        """
-        Test the parser without having to execute the code
-        """
-        args = ['-f', '/repo/embargo',
-                '-t', '/home/j/jarugula/scratch',
-                '--instrument', 'LATISS',
-                '--embargohours', '80.0',
-                '--datasettype', 'raw',
-                '--collections', 'LATISS/raw/all']
-        parser = move_embargo_args.parser(args)
-        self.assertTrue(parser.long)
-        '''
-
-        #python move_embargo_args.py --help
-        #python move_embargo_args.py -fromrepo $arg_fromrepo
-            -torepo $arg_torepo
-            -instrument $arg_instrument
-            -days $arg_days
-            -dtype $arg_dtype
-        self.assertRaises(move_embargo_args())
-        self.assertRaises(move_embargo_args("-fromrepo" = arg_fromrepo,
-                                            "-torepo" = arg_torepo,
-                                            "-instrument" = arg_instrument,
-                                            "--embargohours" = arg_hours,
-                                            "-dtype" = arg_dtype))
-        '''
-
-
-if __name__ == '__main__':
+import shutil
+import os
+import tempfile
+
+from lsst.daf.butler import Butler
+
+
+def is_it_there(
+    embargo_hours: float,
+    now_time_embargo: str,
+    ids_should_remain_after_move,
+    ids_should_be_moved,
+    temp_from,
+    temp_to,
+    move,
+):
+    # Run the package
+    subprocess.call(
+        [
+            "python",
+            "../src/move_embargo_args.py",
+            "-f",
+            temp_from,
+            "-t",
+            temp_to,
+            "--embargohours",
+            str(embargo_hours),
+            "--instrument",
+            "LATISS",
+            "--datasettype",
+            "raw",
+            "--collections",
+            "LATISS/raw/all",
+            "--nowtime",
+            now_time_embargo,
+            "--move",
+            move,
+        ]
+    )
+    # first test stuff in the temp_to butler
+    butler_to = Butler(temp_to)
+    registry_to = butler_to.registry
+    ids_in_temp_to = [
+        dt.dataId.full["exposure"]
+        for dt in registry_to.queryDatasets(datasetType=..., collections=...)
+    ]
+    print(ids_in_temp_to)
+    # verifying the contents of the temp_to butler
+    # check that what we expect to move (ids_should_be_moved)
+    # are in the temp_to repo (ids_in_temp_to)
+    for ID in ids_should_be_moved:
```

Comment:
Can you use `assert ids_should_be_moved == ids_in_temp_to`?  `pytest` is pretty good about highlighting the differences on failure.

Reply:
I've simplified these assertion for statements also using a sorted().